### DISCUSSION TITLE
Is there a way to get the status label value of an item in a script

### DISCUSSION THREAD

**Discussion by atownson** [netbox-community/netbox#17179] [Help Wanted!] [answered]
Hey everyone,
I've written a script that outputs some of the instantiated models and their field values. One of the fields is status. The database stores the value (first tuple item) of the corresponding choice set for the model. Is there a way to resolve the status back to the label rather than the value without hard-coding it?

For example, I'd rather return `Active` than `active`.

As a note, we do use custom status values in the configuration if that matters.

Thank you.

**Comment by zmaster7** [comment, marked as answer]
It looks like each model with the status field has a get_status_display() option, which will return the label for the status it has, e.g:

`Device.objects.first().get_status_display()`

In addition to get_status_display(), get_status_color() is another option.

On a side note, utilizing [nbshell](https://netboxlabs.com/docs/netbox/en/stable/administration/netbox-shell/) can be helpful to determine what options are available for a given model.

**Comment by atownson** [comment]
`get_status_display()` works great. Thanks @zmaster7.
